tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -3,55 +3,26 @@
 
 import json
 import os
-import shutil
 from decimal import Decimal
-from tempfile import mkdtemp
 from typing import Any, Tuple
 
 import pytest
 from pyspark.sql import SparkSession
 from pyspark.sql.functions import col
 from pyspark.sql.types import DecimalType, IntegerType, StringType, StructField, StructType
 
-from streaming import MDSWriter
 from streaming.base.converters import dataframeToMDS
 
 MY_PREFIX = 'train'
-MY_BUCKET = 'mosaicml-composer-tests'
-MANUAL_INTEGRATION_TEST = False
+MY_BUCKET = {
```

Comment:
It looks like `MY_BUCKET` and `MY_PREFIX` is not used in this file. Can we please remove it?

Reply:
Done.